eng/scripts/automation/Invoke-CadlDataPlaneGenerateSDKPackage.ps1

```diff
@@ -3,16 +3,10 @@
 .SYNOPSIS
 script for creating a getting started project in a branch of `azure-sdk-for-net` repo.
 
-.PARAMETER service
-The Azure client service directory name. ie. purview. It is the same as the name of the directory in the specification folder of the azure-rest-api-specs repo that contains the REST API definition file.
+.PARAMETER sdkFolder
```

Comment:
we should do nothing if the folder structure already exists.

Reply:
Yes.